backend/src/main/java/gov/cdc/usds/simplereport/service/errors/InvalidAddressException.java

```diff
@@ -0,0 +1,11 @@
+package gov.cdc.usds.simplereport.service.errors;
+
+import java.io.Serial;
+
+public class InvalidAddressException extends RuntimeException {
+  @Serial private static final long serialVersionUID = 1L;
```

Comment:
I am not familiar with @Serial, what does it do?

Reply:
That particular `@Serial` annotation was recommended by Intellij to allow compile-time checking of serialization-related declarations, similar to the checking enabled by the `@Override` annotation type to validate method overriding